backend/utils/AbsolutePaths.ts

```diff
@@ -0,0 +1,34 @@
+import path from 'path';
```

Comment:
```suggestion
import path from 'node:path';
```

Reply:
What is the difference here? Or is this just for documentation that we use the node path package?